eng/native/configurecompiler.cmake

```diff
@@ -781,6 +781,8 @@ if (CLR_CMAKE_HOST_WIN32)
       set(CMAKE_ASM_MASM_COMPILE_OPTIONS_MSVC_RUNTIME_LIBRARY_MultiThreadedDLL      "")
       set(CMAKE_ASM_MASM_COMPILE_OPTIONS_MSVC_RUNTIME_LIBRARY_MultiThreadedDebug    "")
       set(CMAKE_ASM_MASM_COMPILE_OPTIONS_MSVC_RUNTIME_LIBRARY_MultiThreadedDebugDLL "")
+      # Generate CodeView debug information.
+      set(CMAKE_ASM_MASM_FLAGS "${CMAKE_ASM_MASM_FLAGS} /Zi")
```

Comment:
Can we use `add_compile_options` here instead of setting the `CMAKE_ASM_MASM_FLAGS` variable directly?

Reply:
Earlier in the file `add_compile_options` is used for C and C++. I tried adding it there and tested that .ASM files and C++ files still have usable symbols.

https://github.com/dotnet/runtime/blob/960e4d723c27a5407dc691d06e546bc455a9c4a5/eng/native/configurecompiler.cmake#L673